### PR TITLE
#3828 enable to dynamically configure the web site of downloading kaf…

### PR DESCRIPTION
**Pull request by chia7712**
…ka/zookeeper binary

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3828)
I have tested it on my local, and build works well.
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docker/broker.dockerfile [hunk 1]
```diff
@@ -26,7 +26,8 @@ RUN yum install -y \
 ARG KAFKA_DIR=/opt/kafka
 ARG KAFKA_VERSION=2.4.0
 ARG SCALA_VERSION=2.12
-RUN wget https://archive.apache.org/dist/kafka/${KAFKA_VERSION}/kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz
+ARG WEB=http://apache.stu.edu.tw
```
Comment (vitojeng) [inline docker/broker.dockerfile]: `MIRROR_SITE` would be more descriptive?

Comment (chia7712) [reply docker/broker.dockerfile]: the https://archive.apache.org/dist/kafka is better to us :)

let me fix it

File: docker/broker.dockerfile [hunk 2]
```diff
@@ -26,7 +26,8 @@ RUN yum install -y \
 ARG KAFKA_DIR=/opt/kafka
 ARG KAFKA_VERSION=2.4.0
 ARG SCALA_VERSION=2.12
-RUN wget https://archive.apache.org/dist/kafka/${KAFKA_VERSION}/kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz
+ARG WEB=https://archive.apache.org/dist
```
Comment (vitojeng) [inline docker/broker.dockerfile]: I mean about the arg naming. ^^
Maybe be use `MIRROR_SITE` would be better than `WEB`?

Comment (chia7712) [reply docker/broker.dockerfile]: got it :)

**Pull request merged**